README.md

```diff
@@ -21,6 +21,55 @@ A flexible and scalable platform for running Kubernetes control plane APIs.
 - Single binary CLI for improved user experience:
     - initialize, install operator, manage lifecycle of control planes and contexts.
 
+
+## Prerequisites
+Before you begin, ensure you have the following tools installed and configured:
+
+- **[kind](https://kind.sigs.k8s.io)**: Required for the default hosting cluster setup.
+- **[kubectl](https://kubernetes.io/docs/tasks/tools/)**: Required for interacting with Kubernetes clusters.
+- **`sudo` command**: The installation script for the `kubeflex` CLI uses `sudo`.
+    - **Linux/macOS**: `sudo` is typically pre-installed.
+    - **Windows**: Windows does not have `sudo` natively. You can use a `sudo` equivalent like gsudo.
+        You can install `gsudo` using one of the following package managers:
+        - **Chocolatey**: `choco install gsudo`
+        - **Scoop**: `scoop install gsudo`
+        - **WinGet**: `winget install gerardog.gsudo`
+        After installing `gsudo`, you can typically use `gsudo` in place of `sudo` in the commands.
+        Alternatively, if you are using Windows Subsystem for Linux (WSL), `sudo` will be available within your WSL distribution.
+
+        **Note**: The `kflex init --create-kind` command relies on Docker to run `kind`. Ensure Docker Desktop (or another Docker provider compatible with `kind`) is installed and running.
```

Comment:
Why is this note nested under "sudo"?
Is the point here that everycommand that uses docker needs to implicitly be prefixed with `sudo`? If so, then (A) that should appear in the commands themselves and (B) that is just one possible approach, while many people take the opposite approach (configure something so that `docker` can be used _without_ `sudo`).